Managing users page (#74)
NHSDigital/record-a-vaccination-guidance: app/managing-users.md

---
layout: page
title: Managing users
analytics_key: managing-users
parent:
  title: How to use the pilot service
  url: /guide
previous:
  title: Reporting
  url: /reporting
next:
  title: Managing vaccines
  url: /managing-vaccines
---

{% include 'guide-contents.njk' %}

## Adding new users  

Only lead administrators can add and manage users. 

To add a user, you must enter their: 

* First name  
* Last name 
* NHS-approved email address  
* Clinician status

## Selecting clinician status 

> Select ‘yes’ if users are registered clinicians. This will allow them to assess patients and record their consent.  

Reviewer: I wonder if this is slightly confusing, as it’s technically about whether they can be recorded as having assessed the patient, rather than whether they're allowed to? Although that's quite a hard thing to explain. Hmm.

Author: Good point, instead 'Select ‘yes’ if users are registered clinicians. As a result, they can be selected as assessing and consenting clinicians.

Reviewer: Nice. Have amended this.

> Select ‘yes’ if users are registered clinicians. This will allow them to assess patients and record their consent.  

Reviewer: ```suggestion
Select ‘yes’ if users are registered clinicians. As a result, they can be selected as assessing and consenting clinicians.
```

## Selecting permission levels 

Choose and select a permission level for new users: 

| **Permission level**               | **Lead administrator**    | **Administrator**    | **Recorder**    |
|------------------------------------|---------------------------|----------------------|-----------------|
| **Records and edits vaccinations** | Yes                       | Yes                  | Yes             |
| **Adds and manages vaccines**      | Yes                       | Yes                  | No              |
| **Creates reports**                | Yes                       | Yes                  | No              |

> | **Adds and manages users**         | Yes                       | No                   | No              |

Reviewer: This confused me at first as I was expecting the permission levels to be below the "Permission level" heading rather than to the right... Might just be me though?

Could try flipping it around so that each permission level is a row, and the columns are the features they can and can't use?

> You can select more than one user for each permission level. 

Reviewer: "Select" feels odd to me here. "have"? or "add"?

```suggestion
You can have more than one user for each permission level. 
```

![Screen shows user adding a new user's details and selecting their permission level](/images/manage-users-add.png)

## Creating a RAVS account  

> Once you have added a new user, we’ll automatically create a RAVS and Okta user account. We'll also send them a RAVS welcome email with instructions to access the service. Okta allows new users to verify their identity and use RAVS.  

Reviewer: This sentence is slightly contradicted by the one below about using an existing Okta account if they have one. Maybe shorten it and leave the description of setting up the Okta account to the 2 bullets below?

Author: Thanks for the spot, agree with that so how about:

Once you have added a new user, we’ll automatically create a RAVS account and send them a welcome email to access the service.  

Okta allows new users to verify their identity and use RAVS.  

- If users already have an Okta account for other NHS applications, then their RAVS account will be added to their existing Okta account. 
- If users are new to Okta, they’ll also receive an Okta email with instructions to set up their account. They’ll need to do this first to access RAVS.

If users:
* already have an Okta account for other NHS applications, their RAVS account will be added to their existing Okta account. 
* are new to Okta, they’ll also receive an Okta email with instructions to activate it. They’ll need to do this first to access RAVS.

![Screen shows user a preview email before they send it](/images/manage-users-check.png)

## Changing users  

You can change user permission levels and clinician status from the 'Manage users' screen.  

If you remove their clinician status, they can no longer assess patients and record their consent.  

If you change their permission levels, this will become active when they log back into the service.  

**Important:** You cannot change your permission level if you’re the only 'Lead administrator' for your organisation. 

![Screen shows user the manage users screen](/images/manage-users-home.png)

## Deleting users  

Delete a user account by emailing [ravs.support@england.nhs.uk](mailto:ravs.support@england.nhs.uk) or by calling **0121 611 0187** (select option 3).  

## Users across multiple organisation  

> To add users to your organisation that are users in another organisation, they'll need a different email address. They can only have one email address per organisation.  

Reviewer: Should this be the other way around?

```suggestion
To add users to your organisation that are users in another organisation, they'll need a different email address. They can only have belong to one organisation per email address.  
```

And should we mention contacting the helpdesk if they need to "move" organisation but keep the same email address?

Author: Just amended to 'They need a unique email address for each organisation.'

Reviewer: Couldn't see your amend (was it in a different branch) so I've amended it directly based on your suggestion, thanks.

> To add users to your organisation that are users in another organisation, they'll need a different email address. They can only have one email address per organisation.  

Reviewer: ```suggestion
To add users to your organisation that are users in another organisation, they’ll need a different email address. They need a unique email address for each organisation
```
